lab-jonathan/test/greet-test.js

```diff
@@ -0,0 +1,24 @@
+var verify = require('../lib/greet.js');
+var assert = require('assert');
+
+
+
+
+describe('app should say hello and give the provided name', function (){
+  describe('greet', function(){
+    it('will say hello to a user', function() {
+      assert.strictEqual('hello Jonathan', verify('Jonathan'))
```

Comment:
You have a mix of semi-colons and not... The linter rule is not checking this, so you want to either use them or don't but be consistent.